File: examples/src/bin/delete_account.rs
use objectscale_client::client::ManagementClient;

fn main() {
    let endpoint = "https://10.225.108.189:443";
    let username = "root";
    let password = "Password123@";
    let insecure = true;

    let account_id = "osai74a971da08113120";

    let mut client =
        ManagementClient::new(endpoint, username, password, insecure).expect("management client");
    client.delete_account(account_id).expect("delete account");
    println!("Deleted account: {}", account_id);
}

File: python/examples/delete_account.py
import objectscale_client

def main():
    endpoint = "https://10.225.108.189:443"
    username = "root"
    password = "Password123@"
    insecure = True

    account_id = "osai8f36c29a17795572"

    try:
        client = objectscale_client.client.ManagementClient(endpoint, username, password, insecure)
        client.delete_account(account_id)
        print("Deleted account:", account_id)
    except Exception as e:
        print("Failed to delete account:", e)


if __name__ == '__main__':
    main()
